### PR TITLE
Cherry-pick: 6.3 packaging version update

### PR DESCRIPTION
**Pull request by stanleytsang-amd**

### CODE REVIEW

**Review by RobsonRLemos — APPROVED** [review]
LGTM.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 Documentation for hipRAND is available at
 [https://rocm.docs.amd.com/projects/hipRAND/en/latest/](https://rocm.docs.amd.com/projects/hipRAND/en/latest/).
 
+## hipRAND-2.11.1 for ROCm 6.2.4
```
Comment (amd-jnovotny) [inline CHANGELOG.md:6]: Should this be 6.3.0?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,12 @@
...
+
+### Additions
```
Comment (amd-jnovotny) [inline CHANGELOG.md]:
```suggestion
### Added
```

File: CHANGELOG.md (hunk 1 above)
Comment (stanleytsang-amd) [reply CHANGELOG.md:6]: There are no updates for ROCm 6.3.0, hence no entry for it.  This is merely to add the 6.2.4 entry into the 6.3 changelog for completeness.

File: CHANGELOG.md (hunk 2 above)
Comment (stanleytsang-amd) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 1 above)
Comment (amd-jnovotny) [reply CHANGELOG.md:6]: Thanks. Understood.

**Review by amd-jnovotny — APPROVED** [review]

**Pull request merged**